examples/docs-examples/examples/outbound/outbound-basic.mjs

```diff
@@ -0,0 +1,33 @@
+import { TaskExecutor } from "@golem-sdk/golem-js";
+import { readFile } from "fs/promises";
+
+const url = "https://ipfs.io/ipfs/bafybeihkoviema7g3gxyt6la7vd5ho32ictqbilu3wnlo3rs7ewhnp7lly";
```

Comment:
Add a comment explaining that this will work thanks to the whitelist and then link the whitelist itself for the reader to see/check.

Reply:
done